controllers/apps/transformer_cluster_deletion.go

```diff
@@ -157,7 +157,6 @@ func kindsForHalt() ([]client.ObjectList, []client.ObjectList) {
 	namespacedKinds, nonNamespacedKinds := kindsForDoNotTerminate()
 	namespacedKindsPlus := []client.ObjectList{
 		&kbappsv1.ComponentList{},
-		&appsv1alpha1.OpsRequestList{},
```

Comment:
replace to opsv1alpha1 ?

Reply:
ops is a top resource for cluster, should not deleted by cluster controller. only remove it